tests/resource_payer_cleos_test.py

```diff
@@ -0,0 +1,739 @@
+#!/usr/bin/env python3
```

Comment:
Is this file supposed to be included in an amqp test PR?

Reply:
removed